src/components/jobs-list/job-summary.js

```diff
@@ -71,8 +71,14 @@ const JobSummary = ({ appName, job }) => {
               <FontAwesomeIcon icon={faClock} size="lg" />
             </div>
             <div className="job-summary__data-list">
-              <RelativeToNow time={job.started} titlePrefix="Start time" />
-              <Duration title="Duration" start={job.started} end={job.ended} />
+              <RelativeToNow time={job.created} titlePrefix="Created time" />
```

Comment:
when jobs now can be queued, should we show both Created and Started time?

Reply:
I don't think the extra visual noise of an extra column is justified for something that has so few use cases… but we should display the relative time from start (as originally), not creation. Maybe something like this:
```jsx
{
  !job.started && (
  <React.Fragment>
    ⌛️ <RelativeToNow time={job.created} titlePrefix="Creation time" />
  </React.Fragment>
)}
{
  job.started && (
  <RelativeToNow time={job.started} titlePrefix="Start time" />
)}
```